src/module/api/index.ts

```diff
@@ -2,10 +2,12 @@ import * as update from './update';
 import * as timeline from './timeline';
 import * as texUpd from './texupd';
 import * as audience from './audience';
+import * as message from './message';
 
 export const API = {
   Audience: audience.Audience,
   Update: update.Update,
   Timeline: timeline.Timeline,
   TexUpd: texUpd.TexUpd,
+  Message: message.Message,
```

Comment:
Seeing this clean structure makes me happy. If this would now be sorted alphabetically. 😍

Reply:
haha!

I have a strange sense of joy seeing this also 🧐